src/Sentry.Bindings.Cocoa/buildTransitive/Sentry.Bindings.Cocoa.targets

```diff
@@ -1,5 +1,20 @@
 <Project>
 
+  <!--
+     Workaround for https://github.com/xamarin/xamarin-macios/issues/15252
+     ensuring that Native AOT compile code does not get the wrong marshal-managed-exceptions
+    -->
+  <Target Name="_SentryMarshalExceptionManagement">
+    <PropertyGroup Condition="'$(PublishAot)' == 'true'">
+      <MtouchExtraArgs>marshal-managed-exceptions:disable</MtouchExtraArgs>
+      <MtouchNoSymbolStrip Condition="$([MSBuild]::GetTargetPlatformIdentifier('$(TargetFramework)')) == 'ios'">True</MtouchNoSymbolStrip>
```

Comment:
This worries me a bit. Lots of users are going to want to strip symbols (to reduce the size of their app). Is this required for the fix to work? What happens if users override this in their csproj file?

Reply:
Thanks for pointing this out. I agree with you here.
Usually, it's not required, but it might have been for a reason if I left it there. But let me test that again to be sure

Reply:
I confirm this can be removed.
I did this because of an issue related to [Xcode 16](https://github.com/xamarin/xamarin-macios/issues/19157) I'm gonna remove this and clarify this in the description